<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgglWindow</class>
 <widget class="QMainWindow" name="AgglWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agglomerate</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="agl_layout_input">
      <item>
       <widget class="QLabel" name="agl_label_input">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Input file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="agl_lineEdit_input">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="agl_pushButton_input">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="agl_comboBox"/>
    </item>
    <item>
     <widget class="QListWidget" name="agl_listWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="agl_textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="agl_layout_output">
      <item>
       <widget class="QLabel" name="agl_label_output">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Output file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="agl_lineEdit_output">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="agl_pushButton_output">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="agl_layout_buttons">
      <item>
       <widget class="QPushButton" name="agl_pushButton_clear">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="agl_pushButton_create">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="agl_pushButton_close">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>agl_lineEdit_input</tabstop>
  <tabstop>agl_pushButton_input</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>agl_comboBox</tabstop>
  <tabstop>agl_listWidget</tabstop>
  <tabstop>agl_textBrowser</tabstop>
  <tabstop>agl_lineEdit_output</tabstop>
  <tabstop>agl_pushButton_output</tabstop>
  <tabstop>agl_pushButton_clear</tabstop>
  <tabstop>agl_pushButton_create</tabstop>
  <tabstop>agl_pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>agl_pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>AgglWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
